charts/deployment-operator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: deployment-operator
 description: creates a new instance of the plural deployment operator
 appVersion: 0.5.6
-version: 0.5.6
+version: 0.5.7
```

Comment:
don't bump this quite yet, because we need it to stay equal to app version to mke deployment operator releases make sense

Reply:
Ye, I wasn't sure abut that. Reverted.